<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form1</class>
 <widget class="QWidget" name="Form1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/images/resources/ss.jpg);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
border:none;
background:transparent;
}</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>20</y>
           <width>171</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: center;
</string>
         </property>
         <property name="text">
          <string>FortiFile</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>100</y>
           <width>221</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#767676;&quot;&gt;Welcome to FortiFile&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>190</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Email address</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>340</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>560</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>New to FortiFile?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="forgot_pass_btn">
         <property name="geometry">
          <rect>
           <x>312</x>
           <y>400</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border:none;
color: rgb(0, 134, 110);
}


QPushButton:hover:!checked{
	color: rgb(0, 56, 46);
}</string>
         </property>
         <property name="text">
          <string>Forgot password?</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="email_input">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>210</y>
           <width>291</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>234</y>
           <width>291</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>384</y>
           <width>291</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="password_input">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>360</y>
           <width>291</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="sign_in_btn">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>450</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
background-color: rgb(0, 134, 110);
border-radius:20</string>
         </property>
         <property name="text">
          <string>Sign in</string>
         </property>
        </widget>
        <widget class="QPushButton" name="create_acc_btn">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>550</y>
           <width>93</width>
           <height>41</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border:none;
color: rgb(0, 134, 110);
}


QPushButton:hover:!checked{
	color: rgb(0, 56, 46);
}</string>
         </property>
         <property name="text">
          <string>Create account</string>
         </property>
        </widget>
        <widget class="Line" name="line_3">
         <property name="geometry">
          <rect>
           <x>285</x>
           <y>570</y>
           <width>85</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>266</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Access code</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="acces_code">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>286</y>
           <width>291</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="Line" name="line_4">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>310</y>
           <width>291</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>25</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resource.qrc">:/icons/resources/logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
